src/pages/index.js

```diff
@@ -49,30 +131,68 @@ const enableValidation = (settings) => {
 
 enableValidation(settings);
 
-// Section class Instance
+/*-----------------------------------------------------------------*/
+/*             Class Instances                          */
+/*-----------------------------------------------------------------*/
+
+// Section instance
 const cardSection = new Section(
   {
-    items: initialCards,
+    items: [],
     renderer: (data) => {
-      cardSection.addItem(createCard(data));
+      cardSection.addItem(createCard(data), "append");
     },
   },
   cardContainer
 );
-cardSection.renderItems();
 
-// UserInfo class instance
-const userInfo = new UserInfo({
-  profileName: "#profile_name",
-  profileJob: "#profile_job",
+// Edit Avatar instance
+const popupWithAvatarForm = new PopupWithForm(
+  "#edit-avatar-modal",
+  (formData) => {
+    savingBtn.textContent = "Saving...";
+    const avatarLink = formData.avatar;
+    api
+      .changeProfilePic(avatarLink)
+      .then((data) => {
+        document.getElementById("profile__avatar").src = data.avatar;
+        popupWithAvatarForm.close();
+      })
+      .catch((err) => console.error(err));
+  }
+);
+
+popupWithAvatarForm.setEventListeners();
+
+changeAvatarbtn.addEventListener("click", () => {
+  // assign them to the value of the corresponding input elements
+  api
+    .getUserInfo()
+    .then((userData) => {
+      avatarInput.value = userData.avatar;
```

Comment:
You don't need to call `api.getUserInfo()` when you open the avatar modal because it's a waste of resources. You'll have this data in `UserInfo`